src/agent/onefuzz-supervisor/src/config.rs

```diff
@@ -111,13 +111,12 @@ impl Registration {
         let mut url = config.register_url();
         url.query_pairs_mut()
             .append_pair("machine_id", &machine_id.to_string())
-            .append_pair("pool_name", &config.pool_name);
+            .append_pair("pool_name", &config.pool_name)
+            .append_pair("version", env!("ONEFUZZ_VERSION"));
 
         if managed {
             let scaleset = onefuzz::machine_id::get_scaleset_name().await?;
-            url.query_pairs_mut()
-                .append_pair("scaleset_id", &scaleset)
-                .append_pair("version", env!("ONEFUZZ_VERSION"));
+            url.query_pairs_mut().append_pair("scaleset_id", &scaleset);
```

Comment:
Is this change related?

Reply:
No.   This PR was split off from a larger one.  That said, this is needed.  Going to leave it here.